.yo-rc.json

```diff
@@ -17,21 +20,21 @@
     "messageBroker": false,
     "serviceDiscoveryType": "consul",
     "buildTool": "gradle",
-    "enableSocialSignIn": false,
-    "jwtSecretKey": "b5838c74b2228da2189310693b41bac7c3162d51",
-    "enableTranslation": true,
-    "applicationType": "microservice",
+    "enableSwaggerCodegen": false,
+    "jwtSecretKey": "MmJjODQ3NmJmYzI0YWIwZmMwNzI1N2IwNmUxZjhmZWJiNDJmN2Y4OTkwNDA5MmYzMzhmMDA0YTY0OWEwYzUyODA2MjU0MzFmZDI2OWIyYzY2OTQyZjAyMTllYzVkYmM2MWViZjkyYjdhOWMzYzNiZDY5MTU4YzM0ZmFjNTkwNDc=",
+    "embeddableLaunchScript": false,
     "testFrameworks": [
       "gatling",
       "cucumber"
     ],
     "jhiPrefix": "jhi",
+    "entitySuffix": "",
+    "dtoSuffix": "DTO",
+    "otherModules": [],
+    "enableTranslation": false,
```

Comment:
enableTranslation was `true`. May it affect the logic?

Reply:
No, because we use the same configuration and classes for working with database